library/foundations/scripts/templates/storybook.template.ts

```diff
@@ -203,11 +197,17 @@ const renderLayerVariables = ({ name, variables }: TokenLayer) => {
     })
     .join('');
 
-  // TODO: fix broken `components--components` typography variables (as well as any other values starting with `{--` as excluded above), they are not being generated correctly and are breaking the entire story. As a workaround, they are being temporarily excluded from generation.
-  return `
+  // TODO: values starting with `--` are not being generated correctly and are therefore not being displayed correctly.
+  // As a workaround, layers containing these variables have been temporarily excluded from generation.
+  return name === 'components--components' ||
+    name === 'mode--light' ||
+    name === 'mode--dark' ||
+    name.startsWith('breakpoint')
```

Comment:
Documentation not generated for everything that is broken, and added BOOST-302 to fix the broken layers/tokens.